homeassistant/components/fan/__init__.py

```diff
@@ -623,7 +623,7 @@ def state_attributes(self) -> dict:
     @property
     def supported_features(self) -> int:
         """Flag supported features."""
-        return 0
+        return self._attr_supported_features
```

Comment:
Fans do not support the short hand attr attributes yet.

Reply:
Ok I see, so the correct fix (for now) would actually be to reinstate the `supported_features` property inside the KNX fan entities?

Reply:
Right!